ng-openapi-gen-schema.json

```diff
@@ -124,12 +124,13 @@
       ]
     },
     "enumStyle": {
-      "description": "Determines how root enums will be generated. Possible values are:\n\n- `alias`: just generate a type alias with the possible values;\n- `upper` for an enum with UPPER_CASE names, and;\n- `pascal` for enum PascalCase names. Defaults to 'pascal'.",
+      "description": "Determines how root enums will be generated. Possible values are:\n\n- `alias`: just generate a type alias with the possible values;\n- `upper` for an enum with UPPER_CASE names;\n- `pascal` for enum PascalCase names, and;\n- `unmodified` to use the value as the name after basic character parsing. Defaults to 'pascal'.",
```

Comment:
@josurzyn Have you had a look at this `alias` option? I can't see where it is implemented, or if at all!. Maybe this option does what we need already?

Reply:
Yea I had a look but it creates a type alias (aka string literal type) rather than an enum. Obviously we could refactor to use string literal type's instead, I assumed that's not the direction we wanted to go, but let's talk about it!